Zapier (#1780)
enkidevs/curriculum: zapier/introduction/zapier-filtering/zapier-what-are-filters.md

---
author: kapnobatai136

category: must-know

aspects:
  - introduction

type: normal

---

# What Are Filters?

---
## Content

Filters represent a very important part of Zapier because they allow you to control which data goes forward in the flow. Basically, if the received information does not fit a certain constraint (defined by you), then the flow won't continue. 

> Adding a filter is as easy as adding a new step, all you have to do is choose `'Filter by Zapier'` as your app.

Reviewer: I don't think it's clear that a) filters are added as steps in the flow and b) to add one you can find them under an app called `Filter by Zapier`

> We have briefly used a filter in the past to check whether a column in our spreadsheet had a certain value or not. One important feature of filters is that they are not limited to one constraint per step.

Reviewer: ```suggestion
We have briefly used a filter in the past to check whether a column in our spreadsheet had a certain value or not. Keep in mind that filters are not limited to one constraint per step.
```

also, would using "check" instead of "constraint" be clearer?

> Going back to the zap we have created, for step 2 (the filter step), let's add another constraint. After clicking on the step card, go to the `'Filter Setup & Testing'` section. It should look something like this:

Reviewer: ```suggestion
Going back to the zap we have created, for step 2 (the filter step), let's see where we could add another constraint. After clicking on the step card, go to the `'Filter Setup & Testing'` section. It should look something like this:
```
